graph/graph_test.go

```diff
@@ -301,6 +301,6 @@ func tempGraph(t *testing.T) (*Graph, graphdriver.Driver) {
 }
 
 func nukeGraph(graph *Graph) {
```

Comment:
just saw this was in a test file, defining a helper function makes more sense than defining a method in that context